### ISSUE TITLE
error due to a missing project settings

### ISSUE THREAD

**Issue by LaurentOngaro** [albinaask/Log#7]
when running my godot project, the following essor message appears in the debugger tab.
Its probably due to missing value in the initial settings for the plugin.

"Invalid type in function 'get_datetime_dict_from_system' in base 'Time'. Cannot convert argument 1 from Nil to bool."

This is the Stack Frames :

O - - at function: _internal_log
1 - res://addons/logger/log-stream.gd:57- at function: info
2 - res://addons/logger/log-stream.gd:201- at function: set level
3 - res://addons/logger/log-stream.gd:48 - at function: init

the code line is 
```
	var now = Time.get_datetime_dict_from_system(ProjectSettings.get_setting(Settings.USE_UTC_TIME_FORMAT_KEY))
```

I fixed the issue be changing the setting for the add on but, in that case, another error occurs

"Invalid call. Nonexistent 'String' constructor."

This is the Stack Frames :

O - res://addons/logger/log-stream.gd:119 - at function: _ internal_log
1 - res://addons/logger/log-stream.gd:57 - at function: info
2 - res://addons/logger/log-stream.gd:201 - at function: _set_level
3 - res://addons/logger/log-stream.gd:48 - at function: _init
4 - res://addons/logger/logger.gd:8 - at function: _init

the code line is 
```
	var msg = String(ProjectSettings.get_setting(Settings.LOG_MESSAGE_FORMAT_KEY)).format(format_data)
```

the version of godot is: Godot Engine v4.2.1 .stable.official [b09f793f51

Runing on windows 11 up to date

**Comment by albinaask** [comment]
Yes! This is a known problem that will hopefully be fixed in the coming days. Use the version 1.0 for now where this bug is not present!

**Comment by albinaask** [comment]
The latest release should fix this issue, and should go live on the asset lib as soon as the pending approval goes through. This usually takes at max a couple of days, but you may use the latest version directly from here.

**Comment by LaurentOngaro** [comment]
It seems to be solved for me with the 1.1.2 release
thanks for your reactivity !
